src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7690,9 +7739,10 @@ private bool IsPossibleLocalDeclarationStatement(bool isGlobalScriptLevel)
 
             tk = this.CurrentToken.ContextualKind;
 
-            var isPossibleAttributeOrModifier = (IsAdditionalLocalFunctionModifier(tk) || tk == SyntaxKind.OpenBracketToken)
-                && (tk is not (SyntaxKind.AsyncKeyword or SyntaxKind.ScopedKeyword) || ShouldContextualKeywordBeTreatedAsModifier(parsingStatementNotDeclaration: true));
-            if (isPossibleAttributeOrModifier)
+            var isPossibleModifier = IsAdditionalLocalFunctionModifier(tk)
+                && (tk is not (SyntaxKind.AsyncKeyword or SyntaxKind.ScopedKeyword)
+                || ShouldContextualKeywordBeTreatedAsModifier(parsingStatementNotDeclaration: true));
```

Comment:
The indentation here is misleading: the `||` is inside the `(`, not at the same level as the `&&`.

Reply:
indeed.  will fixup.